Remotion/Web/ClientScript/Scripts/Utilities.ts

```diff
@@ -200,6 +200,52 @@ class ArgumentUtility
     ArgumentUtility.CheckNotNull(name, value);
     ArgumentUtility.CheckTypeIsJQueryObject(name, value);
   };
+
+  // Resolves any provided css selector, checks the element for null and returns it.
+  public static ResolveCssSelectorOrElement<TElement extends Element> (selectorOrElement: CssSelectorOrElement<TElement>, context?: ParentNode): TElement
+  {
+    if (TypeUtility.IsString(selectorOrElement))
+    {
+      const queryContext = context ? context : document;
+      const resolvedElement = queryContext.querySelector(selectorOrElement) as TElement;
+      if (!resolvedElement)
+        throw ('Error: Cannot find an element specified by selector "' + selectorOrElement + '".');
+
+      return resolvedElement;
+    }
+    else if (TypeUtility.IsObject(selectorOrElement))
+    {
+      ArgumentUtility.CheckNotNull("selectorOrElement", selectorOrElement);
+      return selectorOrElement;
+    }
+    else
+    {
+      throw ('Error: The type of parameter "selectorOrElements" is "' + (typeof selectorOrElement) + '" but "string" or "object" was expected.');
```

Comment:
Should there be a special case for NULL?

Reply:
`null` is handled in the `object` case. It is important to remember that in JS `typeof undefined === "undefined"` but `typeof null === "object"`. So `null` is just a special case of object. If a value is `=== typeof "string"` it cannot be `null`. In this case the null value is therefore handled in the second case.

Reply:
I was thinking in terms of argument checking  / providing a clear error message that something's NULL. It may be that `(typeof NULL)` already prints something kind of OK, but the argument utility is more precise.

Reply:
I think I did not explain my point well enough. A `null` value won't ever be handled in this `else` branch. This is because `null` has the type `object` and is already handled in the `else if (TypeUtility.IsObject(selectorOrElements))` case. The `ArgumentUtility` is used there to check for `null` values.

Reply:
Ah, okay. And since the Resolve-method already does not accept nullable as long as you're typesafe, there's no issue here.